src/corelibs/U2View/src/ov_msa/phy_tree/MSAEditorTreeViewer.cpp

```diff
@@ -100,11 +103,12 @@ QWidget* MSAEditorTreeViewer::createWidget() {
     MaCollapseModel* collapseModel = editor->getCollapseModel();
     connect(collapseModel, SIGNAL(si_toggled()), this, SLOT(sl_alignmentCollapseModelChanged()));
 
-    MSAEditorSequenceArea* msaSequenceArea = editor->getUI()->getSequenceArea();
-    connect(msaSequenceArea, SIGNAL(si_visibleRangeChanged(QStringList, int)), msaTreeViewerUi, SLOT(sl_onVisibleRangeChanged(const QStringList&, int)));
-    connect(msaSequenceArea, SIGNAL(si_selectionChanged(const QStringList&)), msaTreeViewerUi, SLOT(sl_selectionChanged(const QStringList&)));
+    MsaEditorWgt *msaEditorUi = qobject_cast<MsaEditorWgt *>(editor->getUI()->getUI());
+    MSAEditorSequenceArea *msaSequenceArea = msaEditorUi->getSequenceArea();
+    connect(msaSequenceArea, SIGNAL(si_visibleRangeChanged(QStringList, int)), msaTreeViewerUi, SLOT(sl_onVisibleRangeChanged(const QStringList &, int)));
```

Comment:
There are no real changes in these lines. Could you please do not reformat sources or run our official clangformat before you commit. It took 2-3 minutes to me to see that there are no changes here.

Reply:
I will try my best

Reply:
You can review your patch for the changes like this. Otherwise you are asking others to repeat the same job multiple times.